pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -2177,6 +2179,9 @@ type BackupSpec struct {
 	// Default is current timestamp.
 	// +optional
 	CommitTs string `json:"commitTs,omitempty"`
+	// Subcommand is the subcommand for BR, such as start, stop, pause etc.
+	// +optional
+	LogSubcommand string `json:"logSubcommand,omitempty"`
```

Comment:
How about mark it with `+kubebuilder:validation:Enum`?

Reply:
Great idea.